### DISCUSSION TITLE
Changing video quality from 'auto' to a fixed quality doesn't select correctly

### DISCUSSION THREAD

**Discussion by albertgustavsson** [f1multiviewer/issue-tracker#35] [Feature requests] [answered]
When opening a stream, `auto` is selected by default. When changing to any other quality setting, the highest is always selected, no matter which is clicked.
For example changing from `auto` to `HD 720p (3Mbit)` when `HD 1080p (6Mbit)` is available, the latter is the one selected.

**Comment by f1multiviewer** [comment, marked as answer]
This will be fixed in the next release!
